### ISSUE TITLE
Feature Request: Add urgent hint when receiving a message

### ISSUE THREAD

**Issue by Trollwut** [saenzramiro/rambox#665]
Hi folks!

I love using your software, but when I'm not on my PC and get a message, I won't be notified, as the window doesn't get an urgent hint.

It would be nice to have an option that enables getting the urgent hint whenever receiving a new message or by-service.

(I'm using i3wm on Arch Linux, if that matters.)

**Comment by saenzramiro** [comment]
I don't understand your request. In Rambox we have native notifications and also, we change the tray icon to red to displays new activity.

**Comment by Trollwut** [comment]
Yes, that's true. :-)

Now the function I'm missing is, that the app itself gets an urgent hint if it receives a message.

When I click on the notification, this happens. But only after I clicked the notification.

So would that be a feature of Rambox or do I have to blame my notification service for not setting an urgent hint automatically?

**Comment by saenzramiro** [comment]
Can you please explain to me what do you mean about "urgent hint"? Can you show me an example with images or video?

**Comment by Trollwut** [comment]
It's when the app wants to say "hey guy, pls focus me, I got something for you!"

In Windows that would be an app flashing in the taskbar. :-)

**Comment by saenzramiro** [comment]
Oh, I see... But having a flashing taskbar every time you have a new message is not annoying for you? That's why we added the unread counter in the taskbar and red icon in the system tray.

I think for me it will be annoying. Maybe we can create an option in the Preferences to enable or disable this.

**Comment by Trollwut** [comment]
Well, I don't use Windows, so it wouldn't be that annoying to me. :-)

That's why I suggested a "per service" option. For example a new mail shouldn't make it flash for me. But I'd like to get noticed if I missed a Telegram message.

Also... could there be a preference switch if I'd rather have an unread message count in the tray than the taskbar?

**Comment by saenzramiro** [comment]
Maybe... I will think about it and see if I can add it in next version.

**Comment by wedens** [comment]
I use tiling wm and I look at workspaces list to see which workspace requires attention. And URGENT hint highlights such workspace.

**Comment by Trollwut** [comment]
Uuuuuh, thanks mate, I'll wait for that release. :D
